libs/designer-ui/src/lib/displayText/index.tsx

```diff
@@ -0,0 +1,24 @@
+import { Link } from '@fluentui/react';
+
+export interface DisplayTextEditorProps {
+  displayTextText: string;
```

Comment:
nit: can we just change this to text? Given that this is already in the displayText Editor, we don't need to let the user know this.